pillar/fluentd/server_operations_qa.sls

```diff
@@ -45,54 +45,81 @@ fluentd:
             - '@type': monitor_agent
             - bind: 127.0.0.1
             - port: 24220
+    - name: fluentd_server_log
```

Comment:
I'm confused about the reason for having the `FLUENT_LOG` label here and in the other SLS file.

Reply:
This is a way to capture fluentd logs (https://docs.fluentd.org/deployment/logging#capture-fluentd-logs)

Reply:
My confusion was because of the fact that you had the label twice. Once here and once in the other config which you moved to the mitx config. I understand it now :)